components/AlumniCarousel.jsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable @next/next/no-img-element */
+import { ALUMNI } from "../data/alumni";
+import { Navigation } from "swiper";
+import { SwiperSlide } from "swiper/react";
+import Carousel from "./Carousel";
+import "swiper/css";
+import "swiper/css/navigation";
+
+const CAROUSEL_CONTAINER = "text-white rounded-lg w-[20rem] h-[37.5rem] shadow-2xl md:w-4/5 swiper-button-white";
+const ALUMNI_TILE =
+  "bg-[#7055FD] w-[20rem] h-[37.5rem] rounded-lg items-center p-5 flex flex-col shadow-2xl md:w-full md:flex-row";
+const ALUMNI_PROFILE = "flex flex-col items-center mx-auto";
+
+const SPACE_BETWEEN_ELEMENTS = 50;
+
+const AlumniTile = ({ name, file, position, prevRole, testimonial }) => {
```

Comment:
Can we maintain padding on resize between the elements?

<img width="1021" alt="Screen Shot 2022-08-21 at 10 56 55 PM" src="https://user-images.githubusercontent.com/47438886/185849272-c5aef08e-c029-448d-8a89-e8ccd9c6981b.png">